.changeset/thin-donuts-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Add `pluralize()` to the catalog kind header component.
```

Comment:
We can be a bit less specific about the method used :) Just hoping that pluralisation is a word that our spellchecker knows 🙏 😅 
```suggestion
Fixed Entity kind pluralisation in the `CatalogKindHeader` component.
```